cob_bringup/robots/cob4-2.xml

```diff
@@ -35,6 +35,10 @@
 				<arg name="robot" value="$(arg robot)"/>
 				<arg name="pc" value="$(arg cob4-2-b1)"/>
 			</include>
+                        <include file="$(find cob_omni_drive_controller)/launch/stuck_detector.launch">
```

Comment:
could be the stuck_detector.launch file moved/copied to cob_bringup/drivers? For the consistency of cob_bringup..